src/backend/db/migrations/1_initial-migrations.sql

```diff
@@ -11,7 +11,11 @@ CREATE TABLE "user" (
 );
 
 -- table for CM's para list will go here
-
+CREATE TABLE "cm_to_para" (
+  relation_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
```

Comment:
```suggestion
```

(for many-to-many tables, we don't need an extra ID--we can make a composite primary key from the two IDs below)